src/components/ItemDetails.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable dot-notation */
+/* eslint-disable quotes */
```

Comment:
Let's remove all disabled Linters, Linting helps to achieve cleaner and more professional code  ✊ 

Kindly check for other occurrences 🙂

Reply:
Done